e2e-tests/cypress/e2e/journeys/insurance/account/password-reset/link-expired/account-password-reset-link-expired.spec.js

```diff
@@ -0,0 +1,92 @@
+import { linkExpiredPage } from '../../../../../pages/insurance/account/password-reset';
+import { PAGES, BUTTONS } from '../../../../../../../content-strings';
+import { INSURANCE_FIELD_IDS } from '../../../../../../../constants/field-ids/insurance';
+import { INSURANCE_ROUTES } from '../../../../../../../constants/routes/insurance';
+import api from '../../../../../../support/api';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.ACCOUNT.PASSWORD_RESET.LINK_EXPIRED;
+
+const {
+  START,
+  ACCOUNT: {
+    PASSWORD_RESET: {
+      NEW_PASSWORD,
+      LINK_EXPIRED,
+    },
+  },
+} = INSURANCE_ROUTES;
+
+const {
+  ACCOUNT: { PASSWORD_RESET_HASH, PASSWORD_RESET_EXPIRY },
+} = INSURANCE_FIELD_IDS;
+
+context('Insurance - Account - Password reset - link expired page', () => {
+  before(() => {
+    cy.navigateToUrl(START);
+    cy.submitEligibilityAndStartAccountCreation();
+    cy.completeAndSubmitCreateAccountForm();
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+  });
+
+  after(() => {
+    cy.deleteAccount();
+  });
+
+  describe(`when a password reset verfication token has expired and exporter navigates to ${NEW_PASSWORD} with the expired token`, () => {
+    let updatedExporter;
+
+    beforeEach(async () => {
+      /**
+       * Get the exporter so that we can use the ID
+       * to update the password reset verification period.
+       */
+      const exporterEmail = Cypress.env('GOV_NOTIFY_EMAIL_RECIPIENT_1');
+
+      const exportersResponse = await api.getExporterByEmail(exporterEmail);
+
+      const { data } = exportersResponse.body;
+
+      const [firstExporter] = data.exporters;
+      const exporter = firstExporter;
+
+      /**
+       * Update the exporter's password reset verification expiry date via the API,
+       * so that we can mimic missing the verification period.
+       */
+      const now = new Date();
+
+      const milliseconds = 300000;
```

Comment:
could we make this a constant?

Reply:
Good shout - can we do this after this PR, there are 4 instances of this so can do all in one go. There's a few tech improvements need to do, can do all in one PR